EDDiscovery/3DMap/FormMap.cs

```diff
@@ -63,29 +62,79 @@ protected override void OnRenderButtonBackground(ToolStripItemRenderEventArgs e)
 
         public bool ShowTril;
 
-        public FormMap(AutoCompleteStringCollection SystemNames)
+        public List<SystemPosition> VisitedSystems { get; set; }
+
+        public SystemClass CenterSystem {
+            get
+            {
+                return _centerSystem;
+            }
+            set
+            {
+                if (value != null && value.HasCoordinate)
+                {
+                    _centerSystem = value;
+                }
+                else
+                {
+                    _centerSystem = null;
+                }
+            }
+        }
+
+        public String CenterSystemName
```

Comment:
Useful for opening the map on a specific system with only a system name to go on.